rust/parquet/src/encodings/encoding.rs

```diff
@@ -731,16 +731,12 @@ impl<T: DataType> Encoder<T> for DeltaBitPackEncoder<T> {
 /// Helper trait to define specific conversions and subtractions when computing deltas
 trait DeltaBitPackEncoderConversion<T: DataType> {
     // Method should panic if type is not supported, otherwise no-op
-    #[inline]
     fn assert_supported_type();
 
-    #[inline]
     fn as_i64(&self, values: &[T::T], index: usize) -> i64;
 
-    #[inline]
```

Comment:
Outdated, and rectified since, so I'm going to mark this as resolved. Note `#[inline]` doesn't work on traits definitions and clippy warns against it. I'm batching such chages increasing clippy compliance till after this PR lands.